thamos/data/defaultThoth.yaml

```diff
@@ -1,30 +1,47 @@
----
 #
-# This default Thoth configuration file placed in a root of a repo. Please adjust values listed below.
+# This is  Thoth's configuration file placed in a root of a repo used by Thamos
+# CLI as well as by Thoth bots. Please adjust values listed below as desired.
 #
-
-# TODO: Adjust Thoth's host.
-host: thoth-user-api.redhat.com
+host: thoth-user-api.redhat.com  # TODO: Adjust Thoth's host.
 tls_verify: true
-# One of testing, stable, latest.
+
+# Recommendation type - one of testing, stable, latest:
 recommendation_type: stable
-runtime_environment:
+
+# Format of requirements file, currently supported is only Pipenv:
+requirements_format: pipenv
+
+# Operating system for which the recommendations should be created:
+operating_system:
+  name: fedora
+  version: '29'
+
+# Hardware information for the recommendation engine:
+hardware_information:
   cpu_family: 6
   cpu_model: 94
+
+# Software configuration of runtime environment:
+runtime_environment:
+  python_version: '3.6'
+  cuda_version: null
+
+#
+# Configuration of bots:
+#
 managers:
   - name: pipfile-requirements
-  - name: update
```

Comment:
no more updates?

Reply:
I think we should depracate it and implement thoth manager that will basically substitute update manager - thoth will issue a new update if there is a new package release and thoth knows the update can be done (recommendation type latest).

Reply:
WDYT?

Reply:
WDYT?

Reply:
sounds good, but I think it should be an 'update recommendation' issued by thoth and that recommendation is handled by thamos (which will update the Pipfile)?!

Reply:
Thamos or kebechet, if the bot was configured (another thoth consumer).